packages/docusaurus-utils/src/tags.ts

```diff
@@ -6,13 +6,36 @@
  */
 
 import _ from 'lodash';
+import logger from '@docusaurus/logger';
 import {normalizeUrl} from './urlUtils';
 
 /** What the user configures. */
 export type Tag = {
+  // TODO isn't it also possibly undefiend ? (label?: string)
```

Comment:
I think it's fine

The Tags File input is `Partial<Tag>` so it's not the Tag type directly. 

And front matter tag object validation ensures both attributes are always provided at once: 

![CleanShot 2024-05-28 at 17 01 34@2x](https://github.com/facebook/docusaurus/assets/749374/65efe40b-26d4-4759-a4e3-a5c3acc3d505)

So whenever Tag is used, label and permalink are always defined (but sometimes inferred from the key)